web/static/js/components/idea_submission_form.jsx

```diff
@@ -26,26 +27,37 @@ class IdeaSubmissionForm extends Component {
     this.setState({ category: event.target.value })
   }
 
+  handleToggleShowCategories(event) {
+    this.setState({ showCategories: !this.state.showCategories })
+  }
+
+  handleToggleChange(event) {
+    this.props.onToggleActionItem()
+    this.handleToggleShowCategories()
+    this.setState({ category: "action-item"})
```

Comment:
hey @tnewell5. think there might be a small bug here. if we're setting the category to `"action-item"` regardless of the `showCategories` state, i think that if someone unchecks the action item toggle, all submitted ideas will go directly to the action items column until the user explicitly changes the select option to a new category. let me pull the branch and play around with it